### PR TITLE
Updated link text for consistency

### PR DESCRIPTION
**Pull request by rolfedh**
Updated the link text for user clarity and consistency.

### CODE REVIEW

**Review by MichalMaler — APPROVED** [review]

**Review by rsvoboda — CHANGES_REQUESTED** [review]
-1 for the change

I would keep it as is for consistency within the guide, `see the official documentation` phrase is used in other places (I count 7 places in https://github.com/quarkusio/quarkus/blob/e535aa62b0021a5324dc4a30348fd617a6b0baa1/docs/src/main/asciidoc/datasource.adoc)

Can you elaborate on "Updated link text for consistency"? consistency with what?

File: docs/src/main/asciidoc/datasource.adoc [hunk 1]
```diff
@@ -810,7 +810,7 @@ hostDescription:: `<host>[:<portnumber>] or address=(host=<host>)[(port=<portnum
 
 Example:: `jdbc:mysql://localhost:3306/test`
 
-For more information, see the link:https://dev.mysql.com/doc/connector-j/en/[official documentation].
+For more information, see the link:https://dev.mysql.com/doc/connector-j/en/[MySQL Connector/J Reference].
```
Comment (jmartisk) [inline docs/src/main/asciidoc/datasource.adoc:813]: It sounds a bit weird that the header behind the link says "MySQL Connector/J Developer Guide", not "Reference"

Comment (rolfedh) [reply docs/src/main/asciidoc/datasource.adoc]: Updated. Thanks.

**Comment by rolfedh** [comment]
> -1 for the change
> 
> I would keep it as is for consistency within the guide, `see the official documentation` phrase is used in other places (I count 7 places in https://github.com/quarkusio/quarkus/blob/e535aa62b0021a5324dc4a30348fd617a6b0baa1/docs/src/main/asciidoc/datasource.adoc)
> 
> Can you elaborate on "Updated link text for consistency"? consistency with what?

Thanks for your feedback @rsvoboda. 

When I first made this change, I hadn’t noticed the many other instances of “official documentation.” My goal was to add clarity by addressing questions like “Whose official documentation?” (MySQL’s) and “What is the subject of this documentation?” (Connector/J Developer Guide).

I agree that this change would benefit from a more comprehensive review and some discussion. With that in mind, I’m closing this PR.

**Pull request closed without merging**